### PR TITLE
Feature/eosio

### PR DESCRIPTION
**Pull request by regalfaith**
Remove a file

### CODE REVIEW

**Review by architectophile — COMMENTED** [review]
This document is not complete.
Complete all the procedures for creating a smart contract.

Can I update this comment?

**Review by architectophile — COMMENTED** [review]
Good. This should be deleted.

File: blockchain/eosio/concepts/kr/blockchain-ethereum-concepts-smart-contract.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Hello World Contract
```
Comment (regalfaith) [inline blockchain/eosio/concepts/kr/blockchain-ethereum-concepts-smart-contract.md:1]: This document should be completed.

**Review by regalfaith — COMMENTED** [review]
Please check all the comments.

File: blockchain/eosio/concepts/kr/blockchain-ethereum-concepts-smart-contract.md (hunk 1 above)
Comment (regalfaith) [reply blockchain/eosio/concepts/kr/blockchain-ethereum-concepts-smart-contract.md:1]: This a reply to the comment.

**Comment by regalfaith** [comment]
1. Please fill out the document.
2. Change the contract name.
3. Create more smart contracts.

**Comment by regalfaith** [comment]
4. Study C++ syntax first.

**Pull request closed without merging**